ts/metrics/system_metrics.py

```diff
@@ -48,14 +49,34 @@ def disk_available():
     data = psutil.disk_usage('/').free / (1024 * 1024 * 1024)  # in GB
     system_metrics.append(Metric('DiskAvailable', data, 'GB', dimension))
 
-
-def collect_all(mod):
+def gpu_utilization():
+    if num_of_gpu <= 0:
+        return
+
+    result = subprocess.run(
```

Comment:
We use https://github.com/rossumai/nvgpu elsewhere in the code to avoid having to use shell commands